core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/YouTubePlayerExt.kt

```diff
@@ -0,0 +1,11 @@
+package com.pierfrancescosoffritti.androidyoutubeplayer.core.player
+
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+
+/** Returns true if the player is muted, false otherwise. */
+suspend fun YouTubePlayer.isMuted(): Boolean = suspendCoroutine { continuation ->
```

Comment:
Use one line

```kotlin
/** Returns true if the player is muted, false otherwise. */
suspend fun YouTubePlayer.isMuted(): Boolean = suspendCoroutine { continuation ->
    isMutedAsync { isMuted -> continuation.resume(isMuted) }
}
```